src/codeu/chat/server/Server.java

```diff
@@ -172,6 +172,14 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+        Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
+        Uuid.SERIALIZER.write(out, info.getVersion());
```

Comment:
have you tested what happens when this writes null? (since it looks like .getVersion() can return null sometimes)